Samples/MessageZooEvents/ZooEventsProducer.Tests/ZooProducerTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using PactNet;
+using PactNet.Infrastructure.Outputters;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ZooEventsProducer.Tests
+{
+    public class ZooProducerTests
+    {
+        private readonly ITestOutputHelper _output;
+
+        public ZooProducerTests(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Fact]
+        public void EnsureEventProducerHonoursPactWithEventConsumer()
+        {
+            //Arrange
+            const string serviceUri = "http://localhost:9222";
+            var config = new PactVerifierConfig
+            {
+                Outputters = new List<IOutput>
+                {
+                    new XUnitOutput(_output)
+                }
+            };
+
+            var builder = WebHost.CreateDefaultBuilder()
+                .UseUrls(serviceUri)
+                .PreferHostingUrls(true)
+                .UseStartup<TestStartup>();
+
+            using (var host = builder.Build())
+            {
+                host.Start();
+            
+                //Act / Assert
+                IPactVerifier pactVerifier = new PactVerifier(config);
+                    pactVerifier
+                        .ServiceProvider("Event API", serviceUri)
+                        .HonoursPactWith("Event API Message Consumer")
```

Comment:
How does this work? You've changed the consumer + provider names

Reply:
They're actually not used for anything, it's just part of the DSL to make it feel nicer. Will update though.